walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,78 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
```

Comment:
importing with a wildcard (*) is not recommended. Can you add all of them one by one?

Reply:
Done!